scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md

```diff
@@ -0,0 +1,85 @@
+# Multi-PoW Mining Simulation Approach
+
+(_The code base is available on GitHub at [**tari-labs/modelling/scenarios/multi_pow_01**](https://github.com/tari-labs/modelling/tree/master/scenarios/multi_pow_01)_)
+
+## Mining
```

Comment:
Maybe add a section/abstract describing the problem and the results to give the reader an idea of the purpose of this document

Reply:
Done